### DISCUSSION TITLE
Language not showing up in repo

### DISCUSSION THREAD

**Discussion by renzozuk** [github-linguist/linguist#6928] [Classification]
I have a Java repo where the language is not being detected: https://github.com/renzozuk/Spatial_Interpolation-Java-Spark
Other Java repos I have do show that as the repo language

### Expected behaviour
Repo gets "Java" tag.

**Comment by lildude** [comment]
Push another change. Sometimes the language analysis job doesn't run on the first push to GitHub as not all servers have a copy of the code.

**Comment by renzozuk** [comment]
The problem was solved, thanks.
